ktor-client/ktor-client-curl/desktop/interop/libcurl.def

```diff
@@ -3,13 +3,20 @@ package = libcurl
 headers = curl/curl.h
 headerFilter = curl/*
 
+compilerOpts = -Idesktop/interop/include
+
 # there is no libz by default installed on windows, so we need to include it statically
 staticLibraries.mingw = libcurl.a libssl.a libcrypto.a libz.a
 compilerOpts.mingw = -DCURL_STATICLIB
+linkerOpts.mingw_x64 = -Ldesktop/interop/lib/mingwX64
```

Comment:
this will not work, as we need to provide another flag to search for static libraries to embed: [libraryPaths](https://kotlinlang.org/docs/native-definition-file.html#include-a-static-library)

Reply:
Mistake on my end here, will fix.